resen/Resen.py

```diff
@@ -864,11 +864,17 @@ def __get_valid_cores(self):
         # for each JSON file in core directory, read in list of cores
         cores = []
         for fn in os.listdir(core_dir):
+
+            if not fn.lower().endswith('json'):
```

Comment:
This allows hidden files (those with a `.` in front) to be picked up. I don't agree that this should be acceptable. I think we can test for this with `startswith('.')`.

Reply:
As @valentic points out, `glob` helps.


    (py37) [asreimer@localhost cores]$ ls -a
    .  ..  cores.json  testing.json  .testing.json.swp

    In [4]: glob.glob('*')                                                                                         
    Out[4]: ['cores.json', 'testing.json']

So replacing the `os.listdir` with `glob.glob` in combination with the `not fn.lower().endswith('json')` seems to be what we need.

Reply:
ok. What about `glob.glob("*.[jJ][sS][oO][nN]")` that way the case doesn't matter.

Reply:
I have pushed a commit with a code that uses glob.